package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fusliez-notes",
-  "version": "0.8.3",
+  "version": "0.9.0",
```

Comment:
This should be done at the end, where every change has been merged, the language branch and all the rest of the features the rest of us have.